### PR TITLE
tickets/DM-43313: Add danish to ts_imsim

### PR DESCRIPTION
**Pull request by jbkalmbach**
Add wep_estimator option to img_closed_loop to enable switching between tie and danish in WEP.

### CODE REVIEW

File: python/lsst/ts/imsim/closed_loop_task.py [hunk 1]
```diff
@@ -335,6 +335,7 @@ def _run_sim(
         turn_off_sky_background: bool = False,
         turn_off_atmosphere: bool = False,
         imsim_log_file: str = "",
+        wep_estimator: str = "tie",
```
Comment (teweitsai) [inline python/lsst/ts/imsim/closed_loop_task.py:338]: How about using the `enum` instead as in the future the machine learning will be supported?

Comment (jbkalmbach) [reply python/lsst/ts/imsim/closed_loop_task.py:338]: Good point. Refactoring to use that.

File: python/lsst/ts/imsim/closed_loop_task.py [hunk 2]
```diff
@@ -1037,8 +1062,12 @@ def run_img(
             Raw seeing in arcsec.
         imsim_log_file : str
             Location to save imsim log output.
+        wep_estimator_method : str
+            Specify the method used to calculate Zernikes in ts_wep.
+            Options are "Tie" or "danish".
```
Comment (suberlak) [inline python/lsst/ts/imsim/closed_loop_task.py:1067]: Is it intentional that "Tie" starts from uppercase, and "danish" does not?  Would it hurt to use "tie" and "danish"?

Comment (suberlak) [reply python/lsst/ts/imsim/closed_loop_task.py:1067]: ok, reading below I think it's just a typo, as `add_argument` has `tie` and `danish` as I suspected

**Review by suberlak — APPROVED** [review]
Looks good, thanks for the update! Do you have a plot showing that both with "tie" and "danish"  the default  loop converges just fine?

File: python/lsst/ts/imsim/closed_loop_task.py (hunk 2 above)
Comment (jbkalmbach) [reply python/lsst/ts/imsim/closed_loop_task.py:1067]: Yeah, typo. Thanks for the catch. Fixed now.

**Comment by jbkalmbach** [comment]
For convergence plots see the document here: https://confluence.lsstcorp.org/display/LTS/WEP+Estimation+Algorithms

**Comment by jbkalmbach** [comment]
Danish successfully converges for LSST FAM, ComCam and CWFS modes.

Here is CWFS as an example (more plots in document in last comment):
image

**Pull request merged**